pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -48,7 +49,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ ncurses readline gmp mpfr expat libipt zlib guile ]
     ++ lib.optional pythonSupport python3
-    ++ lib.optional doCheck dejagnu;
+    ++ lib.optional doCheck dejagnu
+    ++ lib.optional debuginfodSupport elfutils;
```

Comment:
that does not make much good if elftools does not provide libdebuginfod here.

Should I do something like:
```
let elfutils = elfutils.override{enableDebuginfod = debuginfodSupport; };
```
 or something?